src/main.c

```diff
@@ -1,25 +1,53 @@
 #include <stdlib.h>
 #include <time.h>
+#include <stdio.h>
 
 #include "../include/raylib.h"
 #include "../include/raymath.h"
 
+#include "tmp_structs.h"
 
 int screenWidth = 800;
-int screenHeight = 480;
-char title[] = "Raylib Template";
+int screenHeight = 800;
+char *title = "Minesweeper-RTS";
```

Comment:
I'm not sure if this is a change I recommend but let's not use a pointer to hold the title of the game. This should be a const pointer, and an array works fine. However, the change of name is welcome.

Reply:
I have changed the pointer back to an array